app/Plugin/EnvSource/Lib/lang.php

```diff
@@ -52,8 +56,9 @@
   'er.envsource.token'          => 'Token error',
   
   // Plugin texts
-  'pl.envsource.mode.dupe'      => 'Duplicate Handling Mode',
-  'pl.envsource.name.unknown'   => 'Unknownname',
-  'pl.envsource.redirect.dupe'  => 'Redirect on Duplicate',
-  'pl.envsource.sorid.desc'     => 'This must be set to an environment variable holding a unique identifier for any authenticated user.'
+  'pl.envsource.mode.dupe'               => 'Duplicate Handling Mode',
+  'pl.envsource.sp.type'                 => 'Multi Value Delimiter',
```

Comment:
The goal here is to document which SP is used for the integration, not the behavior of that SP. So this field should be labeled something like "Web Server SP Provider".

Reply:
Done